starter_code/queries.md

```diff
@@ -4,80 +4,98 @@
 
 ### 1. All the companies that it's name match 'Babelgum'. Retrieve only their `name` field.
 
-<!-- Your Code Goes Here -->
+FILTER: {name: {$eq:"Babelgum"}}
+PROJECT: {name:1}
```

Comment:
Yes, and if you donn't want the id field, you have to say it explicitly : Project: {name: 1, _id: 0}